lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -111,14 +111,15 @@ spec = do
             , "  -h,--help                Show this help text"
             , ""
             , "Available commands:"
-            , "  mnemonic                 "
-            , "  wallet                   "
-            , "  transaction              "
-            , "  address                  "
+            , "  mnemonic                 Tooling around mnemonic words."
```

Comment:
This phrasing seems a bit inconsistent.

The other descriptions are all **actions** that one can perform:

* "Manage wallets"
* "Manage transactions"
* "Manage addresses"

But "Tooling around ... " is not an action, which makes it inconsistent.

How about:

* "Generate mnemonics."

Reply:
Originally it was `Generate mnemonic words.` but as pointed [here](https://github.com/input-output-hk/cardano-wallet/pull/690#issuecomment-529455047) it doesn't have to be only about generating, so maybe let's go for `Manage mnemonic phrases.`.

Reply:
That sounds like a fair compromise!